extract.py

```diff
@@ -105,8 +96,7 @@ def clean_zip(household_zip):
 
 
 def clean_dob_fromstr(dob_str, date_format):
-    norm_str = unicodedata.normalize("NFKD", dob_str).encode("ascii", "ignore")
-    return strftime("%Y-%m-%d", strptime(norm_str.decode("ascii"), date_format))
+    return strftime("%Y-%m-%d", strptime(dob_str, date_format))
```

Comment:
Any reason this string is no longer cleaned/normalized?

Reply:
@dehall @radamson the reason that string is no longer cleaned there is that it is already cleaned when it is looked up via `translation_lookup()`. Leaving it in `clean_dob_fromstr()` would be redundant, as when `clean_dob_fromstr()` is called, it is called (on line 180) and passed a str that has been returned from `translation_lookup()` and thus would have already have had `clean_string()` run on it.